app/src/main/res/values-en/strings.xml

```diff
@@ -925,12 +925,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_kerb_height_lowered_ramp">Curb ramp</string>
     <string name="quest_kerb_height_no">No curb at all</string>
 
-    <string name="quest_lanes_title">"How many lanes for cars does this road have?"</string>
+    <string name="quest_lanes_title">"How many car lanes does this road have?"</string>
```

Comment:
See [`lanes=*`](https://wiki.openstreetmap.org/wiki/Key:lanes): all through-traffic full-width lanes.

Reply:
Note that this page suggests that turn lanes are not normally included, but this is incorrect: [turn lanes are virtually always included](https://wiki.openstreetmap.org/wiki/Talk:Key:lanes#Inclusion_of_turn_lanes).

Reply:
To me car lanes is ambiguous at best, and confusing at worst, although lanes for cars isn't much better, but isn't there a hint or something normally?

Indeed having checked the quest it says to leave a note if there are specific bus ones, so maybe my comment is irrelevant.

Reply:
Perhaps "How many vehicle lanes does this road have?", as @peternewman mentions otherwise people might exclude bus or truck lanes from the counts.

Reply:
"Vehicle lanes" seems perfectly understandable to me.

Reply:
hm, a bicycle is not considered a vehicle?

Reply:
But anyway, see @peternewman , it's okay to use "car lanes".

Reply:
> hm, a bicycle is not considered a vehicle?

Legally, it varies by jurisdiction, not dialect. But to a layperson using StreetComplete, [who knows](https://blogs.law.nyu.edu/lifeatnyulaw/when-is-a-bicycle-a-vehicle-and-other-legal-questions/)? Best to avoid this quagmire.

Reply:
Alright, since what lanes exactly are meant is explained when selecting the number of lanes, this will do:

```suggestion
    <string name="quest_lanes_title">"How many lanes does this road have?"</string>
```